src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write code here
+function sortList(list) {
+  const itemsArray = Array.from(list.children);
+
+  itemsArray.sort(
```

Comment:
- `itemsArray` is a bad name, a variable name should be descriptive
- You can use methods in a chain, instead of putting them from the new line